src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx

```diff
@@ -321,6 +324,13 @@
   <data name="PackageLoad_WaitMessage" xml:space="preserve">
     <value>Loading the Acuminator package</value>
   </data>
+  <data name="Setting_BannedAPI_AllowedApiFilePath_Description" xml:space="preserve">
+    <value>A path to the file with allowed APIs that will not be reported by Acuminator even if their containing namespaces or types are contained in the "Banned API File" as banned.
```

Comment:
can we edit this message?
... even if their containing namespaces or types are listed in the "Banned API File" as banned.

Reply:
done